src/autorest-core/app.ts

```diff
@@ -130,7 +130,7 @@ function parseArgs(autorestArgs: string[]): CommandLineArgs {
   };
 
   for (const arg of autorestArgs) {
-    const match = /^--([^=]+)(=([^=]+))?$/g.exec(arg);
+    const match = /^--([^=]+)(=(.+))?$/g.exec(arg);
```

Comment:
allowing `=` in value part of CLI arg, e.g. `--input-file=https://asd/qwe.json?token=flubber` will be parsed correctly. The key/value splitting will always happen at the first `=` which should be fine as keys should never contain that character.